mutation/mutation_partition.cc

```diff
@@ -1876,8 +1876,8 @@ mutation_querier::mutation_querier(const schema& s, query::result::partition_wri
                                    query::result_memory_accounter& memory_accounter)
```

Comment:
is that one in rows_entry move ctor a false positive?

Reply:
it is. i actually checked it before sending this pull request. but i don't yet have a sound workaround / fix to silence it.